timeSeriesUtilities/info.json

```diff
@@ -0,0 +1,221 @@
+{
+    "name": "timeSeriesUtilities",
+    "version": "1.0.0",
+    "label": "Time Series Chart Utilities",
+    "description": "This connector provides utility functions for use with the Time Series Chart Widget",
+    "publisher": "Fortinet",
+    "icon_small_name": {
```

Comment:
Something is not correct here. We will need to correct small json and large icon tag